src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -4408,9 +4427,15 @@ private void AddSynthesizedTupleMembersIfNecessary(MembersAndInitializersBuilder
                                                                                       initializer,
                                                                                       this,
                                                                                       DeclarationModifiers.Private | (property.IsStatic ? DeclarationModifiers.Static : 0),
-                                                                                      backingField);
+                                                                                      backingField); // PROTOTYPE(semi-auto-props): Is field-keyword BackingField needed here?
                                     }
 
+                                    // PROTOTYPE(semi-auto-props):
+                                    // For initializers, we must add "BackingField".
+                                    // "NonFieldKeywordBackingField" makes no sense here.
```

Comment:
>// "NonFieldKeywordBackingField" makes no sense here. [](http://example.com/codeflow?start=36&length=53)

This prototype comment is probably going to be addressed by the changes suggested above around the ```backingField``` local.